api/include/mfxstructures.h

```diff
@@ -1632,7 +1728,7 @@ enum {
 
 #endif
 
-#pragma pack(push, 4)
+MFX_PACK_BEGIN_USUAL_STRUCT()
```

Comment:
`mfxU64          GeneralConstraintFlags;`

If that's to preserve old alignment, I suggest to provide explicit comment in the code before the struct.

Reply:
ok